bin/PythonApps/EPLaunch.app/Contents/MacOS/EPLaunch

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+SCRIPT_DIR="$( cd -- "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 ; pwd -P )"
+echo "Script Path: $SCRIPT_DIR"
+EP_ROOT_DIR=$(dirname "$(dirname "$(dirname "$SCRIPT_DIR")")")
+if [[ "$DEBUG" = 1 ]]; then
+  result=`osascript -e "display dialog \"Script Path: $SCRIPT_DIR\nEP_ROOT_DIR: $EP_ROOT_DIR\" buttons {\"Ok\"}"`
+fi;
+
+EP_CLI=$EP_ROOT_DIR/energyplus
+if [ -f "$EP_CLI" ]; then
+  $EP_CLI auxiliary eplaunch
+else
+  ICON_PATH=$(dirname $SCRIPT_DIR)/Resources/Icon.icns
+  osascript -e 'beep' -e "display dialog \"Could not locate energyplus CLI at\n'$EP_CLI'\" with title \"⚠️  Error\" buttons {\"OK\"} default button \"OK\" with icon POSIX file \"$ICON_PATH\""
+fi
```

Comment:
Add error handling in case E+ isn't found. This shouldn't happen in the install directory unless you do something weird (like deleting energyplus, or trying to MOVE that EPLaunch.app or VersionUpdater.app somewhere else).

But if you run it from the ./bin/PythonApps it logically can't find ../../../energyplus.

<img width="423" alt="image" src="https://github.com/user-attachments/assets/9c6c472a-71c3-4a3d-8dd3-f0f9098e4733" />